components/resume/Header.tsx

```
@@ -1,70 +1,90 @@
-        <button
-          className="text-white hover:bg-blue-400 rounded px-4 mx-1 text-base cursor-pointer"
-          onClick={handleOpenDialog}
-        >
-          🛎 Butler
+      </Link>
+
+      {/* Right side - Action buttons and icons */}
+      <div className="flex items-center space-x-3">
+        {/* Home button */}
+        <Link href="/">
+          <button className="text-white hover:bg-[hsl(203,95%,35%)] rounded p-2 transition-colors">
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)